### PR TITLE
[AArch64][NFC] Switch to LiveRegUnits

### PR DESCRIPTION
**Pull request by LumioseSil**

### CODE REVIEW

File: llvm/lib/Target/AArch64/AArch64FrameLowering.cpp [hunk 1]
```diff
@@ -197,6 +197,7 @@
 #include "llvm/ADT/SmallVector.h"
 #include "llvm/ADT/Statistic.h"
 #include "llvm/CodeGen/LivePhysRegs.h"
```
Comment (jayfoad) [inline llvm/lib/Target/AArch64/AArch64FrameLowering.cpp:199]: Can you remove this include?

Comment (AtariDreams) [reply llvm/lib/Target/AArch64/AArch64FrameLowering.cpp:199]: Does not compile when I do that

Comment (jayfoad) [reply llvm/lib/Target/AArch64/AArch64FrameLowering.cpp:199]: Can you explain why? I don't see any other uses of `LivePhysRegs` in this file. What is it relying on?

Comment (AtariDreams) [reply llvm/lib/Target/AArch64/AArch64FrameLowering.cpp:199]: > Can you explain why? I don't see any other uses of `LivePhysRegs` in this file. What is it relying on?

computeLiveIns method.

**Comment by jayfoad** [comment]
No objection from me but you should get review from an AArch64 maintainer.

**Pull request merged**